src/components/AcceptAll.tsx

```diff
@@ -29,16 +29,21 @@ export function AcceptAll({
   };
 
   return (
-    <div className="column-content">
+    <div className="column-content" role="none">
```

Comment:
certain divs get role `generic` under certain circumstances (tbh i havent looked into exactly why this happens) but it adds an extra totally unnecessary nesting to the accessibility tree. any `role="none"` attributes youre seeing me add is because of that